src/main/java/org/folio/edge/client/InnReachClientProxy.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.edge.client;
+
+import java.io.IOException;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSocketFactory;
+
+import feign.Client;
+import feign.Request;
+import feign.Response;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
```

Comment:
Shoudn't it be @Log4j2 ?

Reply:
Yes again) forgot the replace it, thanks!